### PR TITLE
images: Add rhel-8-5

### PR DESCRIPTION
**Pull request by skobyda**
* [x] image-refresh rhel-8-5

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-02.
Log: https://logs.cockpit-project.org/logs/image-refresh-1825-20210319-151818/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-1825-20210319-151818/

**Comment by marusak** [comment]
FTR you can do `./test-create rhel-8-5` locally to test it out. It speeds process of creating of a new image rapidly, especially in those first steps.

**Comment by martinpitt** [comment]
It's really `./image-create -sv rhel-8-5`. The `-sv` will increase verbosity and allow you to ssh in on failure, so that you can interactively investigate the failing step.

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-01.
Log: https://logs.cockpit-project.org/logs/image-refresh-1825-20210325-130101/

**Comment by cockpituous** [comment]
image-refresh rhel-8-5 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-5-20210325-132222

**Comment by marusak** [comment]
Please first pick up the refreshed image push it here and run tests. Very likely you will need to also send PR into cockpit with test adjustments.

**Comment by marusak** [comment]
Also please add 8-5 into `image-trigger` and also move it into master in testmap. Thanks!

**Comment by marusak** [comment]
> rhel-8-5 — Tests passed 

That is invalid context in this sense. You need to specify repo. Please do either of:
1. `./bots/tests-trigger --repo cockpit-project/bots 1825 rhel-8-5@cockpit-project/cockpit`
2. `./bots/tests-trigger --repo cockpit-project/bots 1825 image:rhel-8-5`

The latter is preferred in general, but in this case even the first would work just fine.

**Comment by marusak** [comment]
> Also please add 8-5 into image-trigger and also move it into master in testmap. Thanks!

Still needs to be done

File: lib/testmap.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
             'fedora-coreos',
             'fedora-33/firefox',
             'rhel-8-4',
+            'rhel-8-5',
```
Comment (marusak) [inline lib/testmap.py:36]: Also please don't leave it hanging in `_manual` when moving to `master`.

Comment (skobyda) [reply lib/testmap.py:36]: Fixed

**Comment by martinpitt** [comment]
Sent https://github.com/cockpit-project/cockpit/pull/15609 . Once this becomes green and lands, we can move the test contexts from _manual to master.

**Comment by martinpitt** [comment]
In that cockpit PR, the [test](https://logs.cockpit-project.org/logs/pull-15609-20210328-094301-0fa4678f-rhel-8-5-bots-1825/log.html) looks rather good -- apparently vdo is broken, but we can investigate this next week. The image clearly works, so let's get it into the weekly refresh routine. Thanks!

**Review by martinpitt — APPROVED** [review]

**Comment by martinpitt** [comment]
Update : vdo is *not* broken, just the test was not marking it as "expect it to work" :grin: cockpit PR updated.

**Pull request merged**